include/ttmlir/Dialect/TTIR/Pipelines/TTIRPipelines.h

```diff
@@ -17,9 +17,10 @@ struct StableHLOToTTIRPipelineOptions
   Option<bool> removeDeadValuesEnabled{
       *this, "enable-remove-dead-values",
       llvm::cl::desc("Enable --remove-dead-values optimization pass."),
-      // Currently this pass fails if module has a name, so keeping the
-      // optimization OFF by default until that issue is fixed on llvm side.
-      llvm::cl::init(false)};
+      llvm::cl::init(true)};
+  Option<bool> sparseConstantPropogationEnabled{
```

Comment:
Nitpick, since you didn't add explanation comment for this option can you please also remove the explanation comment above `removeDeadValuesEnabled` option definition? It doesn't add much value since it is already described in `llvm::cl::desc` parameter. I wish we could have good explanation comments here, but I don't see an easy way to describe these optimizations, people will probably resort to google and llvm documentation anyways.